cc33calibrator/suite/trixie/debian/copyright

```diff
@@ -0,0 +1,96 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: cc33xx-utils
+Upstream-Contact: <preferred name and address to reach the upstream project>
+Source: https://git.ti.com/cgit/cc33xx-wlan/cc33xx-utils
+
+Files:     cc33calibrator/Makefile
+           cc33calibrator/README
+           cc33calibrator/calibrator.c
+           cc33calibrator/calibrator.h
+           cc33calibrator/plt.h
+           cc33conf/Makefile
+           cc33conf/cc33xx-conf.conf
+           cc33conf/cc33xx-conf.ini
+           cc33conf/cc33xxconf
+           cc33conf/crc32.o
+           cc33conf/default.conf
+           cc33conf/dictionary.txt
+           cc33conf/main.o
+Copyright: __NO_COPYRIGHT_NOR_LICENSE__
+License:   __NO_COPYRIGHT_NOR_LICENSE__
+
+Files:     cc33calibrator/cc33xx_debug.c
+           cc33calibrator/cc33xx_debug.h
+           cc33calibrator/cc33xx_plt.c
+           cc33calibrator/cc33xx_plt.h
+           cc33conf/cc33conf.h
+           cc33conf/crc32.c
+           cc33conf/crc32.h
+           cc33conf/main.c
+Copyright: 2011-2021 Texas Instruments
+           2012 Texas Instruments Inc.
+License:   GPL-2.0
```

Comment:
Doesn't seem right.. [COPYING](https://git.ti.com/cgit/cc33xx-wlan/cc33xx-utils/tree/cc33conf/COPYING) file looks like some kinda BSD, but most of the source files have a GPL header. Could you reach out internally to get some clarification on the licence here?

Reply:
I have reached out. Will update accordingly.

Reply:
I've gone and fixed up the licencing mess: https://git.ti.com/cgit/cc33xx-wlan/cc33xx-utils/commit/?id=34e2cf44e9468bb6308bf163da4cf7349f1717ec

Should now be a simple BSD-3 for the whole project.